commands/shared.go

```diff
@@ -97,3 +97,37 @@ func displayNetworksHardforkTimestamps() {
 		log.Warnf("⚠️  There was an error while getting hardfork configs. Error: %v", err)
 	}
 }
+
+func installClientConfigFiles(override bool) {
+	log.Info("⬇️  Downloading geth configuration files...")
+	_ = installConfigGroup(configs.GethConfigDependencies, override)
+	log.Info("✅  Geth configuration files downloaded!\n\n")
+
+	log.Info("⬇️  Downloading erigon configuration files...")
+	_ = installConfigGroup(configs.ErigonConfigDependencies, override)
+	log.Info("✅  Erigon configuration files downloaded!\n\n")
+
+	log.Info("⬇️  Downloading nethermind configuration files...")
+	_ = installConfigGroup(configs.NethermindConfigDependencies, override)
+	log.Info("✅  Nethermind configuration files downloaded!\n\n")
+
+	log.Info("⬇️  Downloading besu configuration files...")
+	_ = installConfigGroup(configs.BesuConfigDependencies, override)
```

Comment:
If the override flag is inside of here. Shouldn't the log messages also come from there?

Reply:
`installConfigGroup` simply calls `Install` for each config in the group, which handles logging by itself - but the configGroup itself doesn't have any data about the client it is installing for, I thought that this 'wrapping' makes it a little clearer, but can change it if that's an issue